tests/test_taling_a_vacation.py

```diff
@@ -0,0 +1,51 @@
+import sys
+
+import pytest
+
+from codecademy.taling_a_vacation import hotel_cost
+from codecademy.taling_a_vacation import plane_ride_cost
+from codecademy.taling_a_vacation import rental_car_cost
+from codecademy.taling_a_vacation import trip_cost
+
+@pytest.mark.parametrize("test_hotel_cost_input,test_hotel_cost_expected", [
+    (-sys.maxsize - 1, (-sys.maxsize - 1)*140),
+    (-sys.maxsize, -sys.maxsize*140),
+    (-1, -140),
+    (0, 0),
+    (1, 140),
+    (3037000499, 425180069860),
+    (sys.maxsize - 1, (sys.maxsize-1)*140),
+    (sys.maxsize, sys.maxsize*140)
+])
+def test_hotel_cost(test_hotel_cost_input, test_hotel_cost_expected):
+    assert hotel_cost(test_hotel_cost_input) == test_hotel_cost_expected
+
+
+@pytest.mark.parametrize("test_plane_ride_cost_input,test_plane_ride_cost_expected", [
+    ("Charlotte", 183),
+    ("Tampa", 220),
+    ("Pittsburgh", 222),
+    ("Los Angeles", 475)
```

Comment:
What about some negative tests? Some unexpected city name? Murmansk? LOL- here you have to expect exception:
https://docs.pytest.org/en/latest/assert.html#assertions-about-expected-exceptions